.github/workflows/fitbot.yml

```diff
@@ -114,6 +115,14 @@ jobs:
                     --thcovmat_if_present
         url=$(vp-upload output/ 2>&1 | grep https)
         echo "REPORT_URL=$url" >> $GITHUB_ENV
+        vp-comparefits $RUNCARD $STABLE_REFERENCE_SET \
+                    --title "Automatic check fit $RUNCARD" \
```

Comment:
Maybe you can updated the title of both reports to include "most recent reference" and "stable reference" or something along those lines